### ISSUE TITLE
README updates

### ISSUE THREAD

**Issue by wekyle** [wemashinc/do#7]
- be more focused on end user
- add bit about inviting bot to a room
- need more clarity on the target api, sequences...
- screenshots would be nice
- developer section: overview of data flow, what reused variable names conceptually represent, etc

**Comment by shockey** [comment]
Overhauled the README, but it's a bit wordy now.

Probably best to keep the most important info in the README and break everything else into another Markdown file.
